catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/config/JdbcConfig.java

```diff
@@ -30,13 +31,12 @@ public class JdbcConfig extends Config {
           .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
           .create();
 
-  public static final ConfigEntry<String> JDBC_DRIVER =
+  public static final ConfigEntry<Optional<String>> JDBC_DRIVER =
```

Comment:
Is this a server config? If not, I think it should be defined in `JdbcCatalogPropertiesMetadata`

Reply:
Why is it also defined here? I think this is unnecessary.

Reply:
I think this goes beyond the scope of the class and may lead to confusion instead. I noticed that @yuqi1129 has already created an issue(#1456) for this.

@jerryshao What's your suggestion?